scripts/build-installer-win64.bat

```diff
@@ -0,0 +1,102 @@
+rem DEPENDENCIES:
+rem   1. Node.js ('npm' binary in PATH)
+rem   2. 7zip    ('7z'  binary in PATH)
+
+set DEFAULT_CARDANO_BRANCH=cardano-sl-0.4
+set MIN_CARDANO_BYTES=50000000
+set LIBRESSL_VERSION=2.5.3
+
+set DAEDALUS_VERSION=%1
```

Comment:
Hmm, maybe not, actually. I see your convenient local build script `windows-build-fresh-daedalus.bat` handles this.